clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -88,9 +87,12 @@ export default async function handler(
   };
   const fidesConfigJSON = JSON.stringify(fidesConfig);
 
-  console.log(
-    "Bundling generic fides.js & Privacy Center configuration together..."
-  );
+  if (process.env.NODE_ENV === "development") {
```

Comment:
we can use `environment.settings.DEBUG` instead to be consistent

Reply:
In this app (privacy center) all the other `console.log` statements are guarded by `process.env` instead of that `DEBUG` flag- that flag is passed along to fides.js options but currently doesn't affect any privacy center logic